src/coverlet.collector/DataCollection/CoverageManager.cs

```diff
@@ -22,24 +22,22 @@ internal class CoverageManager
 
         public IReporter[] Reporters { get; }
 
-        public CoverageManager(CoverletSettings settings, TestPlatformEqtTrace eqtTrace, TestPlatformLogger logger, ICoverageWrapper coverageWrapper,
-                               IInstrumentationHelper instrumentationHelper, IFileSystem fileSystem, ISourceRootTranslator sourceRootTranslator)
+        public CoverageManager(CoverletSettings settings, TestPlatformEqtTrace eqtTrace, ILogger logger,
```

Comment:
Please revert update to this file and keep master alignment.

Reply:
Done.